catalog/imaging/imaging-transformer-nitf/src/main/java/org/codice/alliance/transformer/nitf/image/NitfPreStoragePlugin.java

```diff
@@ -208,7 +208,8 @@ private ContentItem createDerivedImage(String id, String qualifier, BufferedImag
     }
 
     private String buildDerivedImageTitle(String title, String qualifier) {
-        String rootFileName = FilenameUtils.getBaseName(title);
+        // filter non-word characters from the filename for file system storage
+        String rootFileName = FilenameUtils.getBaseName(title).replaceAll("[\\W]", "");
```

Comment:
Handled the case of null or empty `FTITLE` to produce a derived filename of `{qualifier}.jpg`. Added unit tests for various permutations (title is null, empty, valid, contains invalid characters, contains all invalid characters, contains only special chars that are non-alphanum/non-human readable).